static/app/components/forms/blankField.tsx

```diff
@@ -1,16 +1,14 @@
 import * as React from 'react';
 
-import Field from 'sentry/components/forms/field';
-
-type Props = Field['props'];
+import Field, {FieldProps} from 'sentry/components/forms/field';
```

Comment:
are these changes related? I'm finding it difficult to review these PRs :/

Reply:
No, sorry :) needs to be rebased because downstream diverged. Let me do it real quick

Reply:
Fixed, sorry about that :)